examples/databricks_plugin/README.md

```diff
@@ -4,52 +4,47 @@
-This section will provide how to use the Databricks Plugin using flytekit python.
+Flyte can be seamlessly integrated with the [Databricks](https://www.databricks.com/) service,
+enabling you to effortlessly submit Spark jobs to the Databricks platform.
 
-## Installation
+## Install the plugin
 
-The flytekit Databricks plugin is bundled into its Spark plugin, so to use, simply run the following:
+The Databricks plugin comes bundled with the Spark plugin.
+To execute it locally, run the following command:
 
-```{eval-rst}
-.. prompt:: bash
-
-    pip install flytekitplugins-spark
+```
+pip install flytekitplugins-spark
 ```
 
-## How to Build Your Dockerfile for Spark on Databricks
+If you intend to run the plugin on the Flyte cluster, you must first set it up on the backend.
+Please refer to the
+{std:ref}`Databricks plugin setup guide <flyte:deployment-plugin-setup-webapi-databricks>`
+for detailed instructions.
 
-Using Spark on Databricks is extremely easy and provides full versioning using the custom-built Spark container. The built container can also execute regular Spark tasks.
-For Spark, the image must use a base image built by Databricks and the workflow code must copy to `/databricks/driver`
+## Run the example on the Flyte cluster
 
-```{literalinclude} ../../../examples/databricks_plugin/Dockerfile
-:emphasize-lines: 20-32
-:language: docker
-:linenos: true
+To run the provided example on the Flyte cluster, use the following command:
+
+```
+pyflyte run --remote \
+  --image ghcr.io/flyteorg/flytecookbook:databricks_plugin-latest \
+  https://raw.githubusercontent.com/flyteorg/flytesnacks/master/examples/databricks_plugin/databricks_plugin/databricks_job.py \
+  my_databricks_job
 ```
 
-## Configuring the backend to get Databricks plugin working
+:::{note}
+Using Spark on Databricks is incredibly simple and offers comprehensive versioning through a
+custom-built Spark container. This built container also facilitates the execution of standard Spark tasks.
 
-1. Make sure to add "databricks" in `tasks.task-plugins.enabled-plugin` in [enabled_plugins.yaml](https://github.com/flyteorg/flyte/blob/master/deployment/sandbox/flyte_generated.yaml#L2296)
-2. Add Databricks access token to Flytepropeller. [here](https://docs.databricks.com/administration-guide/access-control/tokens.html#enable-or-disable-token-based-authentication-for-the-workspace) to see more detail to create Databricks access token.
+To utilize Spark, the image should employ a base image provided by Databricks,
+and the workflow code must be copied to `/databricks/driver`.
 
-```bash
-kubectl edit secret -n flyte flyte-propeller-auth
+```{literalinclude} ../../../examples/databricks_plugin/Dockerfile
+:language: docker
+:emphasize-lines: 1,7-8,20
 ```
 
-Configuration will be like below
-
-```bash
-apiVersion: v1
-data:
-  FLYTE_DATABRICKS_API_TOKEN: <ACCESS_TOKEN>
```

Comment:
Why do we remove it

Reply:
I'm omitting the instructions because they are already covered in the plugin setup guide that I'm referring to in the README.